cmd/lotus/daemon.go

```diff
@@ -153,6 +154,18 @@ var DaemonCmd = &cli.Command{
 			Name:  "restore-config",
 			Usage: "config file to use when restoring from backup",
 		},
+		&cli.StringFlag{
+			Name:  "trace-to-json",
+			Usage: "starts tracer and outputs to json file defined with this flag",
+		},
+		&cli.StringFlag{
+			Name:  "trace-to-elasticsearch",
+			Usage: "starts tracer and outputs to elasticsearch, flag must contain connection string for elasticsearch",
+		},
+		&cli.StringFlag{
+			Name:  "trace-source-auth",
+			Usage: "auth token for trusted source of traces",
+		},
```

Comment:
We should use the node config instead of defining special flags for this. If node operators really want to pass those through the CLI, we have envconfig - https://github.com/filecoin-project/lotus/blob/964df1a2ecc38bf5ab721b7bb6a03e603eb1c98e/node/config/load.go#L41 - https://github.com/kelseyhightower/envconfig

Reply:
I agree with you, I removed CLI flags (and we already had all these prop. as part of node config)